satellite/console/consoleweb/server.go

```diff
@@ -193,10 +193,16 @@ func (server *Server) bucketUsageReportHandler(w http.ResponseWriter, r *http.Re
 	var projectID *uuid.UUID
 	var since, before time.Time
 
-	tokenCookie, err := r.Cookie("tokenKey")
+	host, _, err := net.SplitHostPort(r.Host)
 	if err != nil {
 		server.log.Error("bucket usage report error", zap.Error(err))
+		server.serveError(w, r, http.StatusNotFound)
```

Comment:
This looks more to me more a `StatusInternalServerError`

Reply:
there is a `TODO` regarding the issue

Reply:
I've seen it after, however, I've seen for unauthenticated, not for this one, so I'm not sure if that's apply considering that you haven't added the `TODO`  , though you may have forgotten it.